tests/functional/backend/corpora/test_api.py

```diff
@@ -244,6 +244,45 @@ def test_dataset_upload_flow_with_atac_seq_dataset(
     )
 
 
+@skip_creation_on_prod
+def test_dataset_reupload_flow_from_manifest(
```

Comment:
Adding this test to test redriving ingesting using the existing manifest. This test case was missing and was also broken without anyone noticing.